### PR TITLE
phpdoc

### PR DESCRIPTION
**Pull request by kendrick-k**
added comments

### CODE REVIEW

**Comment by oliverwoodings** [comment]
Hi @kendrick-k,

Thanks for this. At Qubit we use 2 spaces for indentations. The PR seems to have switched everything to tabs/4 spaces. Need to get this changed back. Other than that, all looks good.

**Comment by kendrick-k** [comment]
Hi should be fixed ! Could be useful to construct different kind of UVObject series eg. for non e-commerce websites, etc.

File: uv-library.php [hunk 1]
```diff
@@ -1,144 +1,198 @@
...
+            window.universal_variable = " . $this->toJSON() . ";
+        </script>
+        <!-- End UV -->
+        \n";
     }
 
-  }
+}
 
-?>
```
Comment (oliverwoodings) [inline uv-library.php]: Seem to have lost the closing php tag

File: uv-library.php [hunk 2]
```diff
@@ -1,144 +1,213 @@
...
+    return "\n
+        <!-- Qubit Universal Variable data layer v" . UV_VERSION . " - PHP Lib v" . UV_LIB_VERSION . " -->
+        <script>
+            window.universal_variable = " . $this->toJSON() . ";
+        </script>
+        <!-- End UV -->
+        \n";
   }
 
-?>
```
Comment (oliverwoodings) [inline uv-library.php]: Seem to have lost the closing php tag

Comment (kendrick-k) [inline uv-library.php]: ah it's useless

**Comment by kendrick-k** [comment]
The closing tag of a PHP block at the end of a file is optional, and in some cases omitting it is helpful when using include() or require(), so unwanted whitespace will not occur at the end of files, and you will still be able to add headers to the response later. It is also handy if you use output buffering, and would not like to see added unwanted whitespace at the end of the parts generated by the included files. (PHPManual)

**Comment by oliverwoodings** [comment]
Ok fair enough, we can leave it as is. I'll merge in a bit.

**Comment by oliverwoodings** [comment]
Actually I've just noticed you've also switched to having opening braces on new lines. Could we please maintain the hanging braces? It's standard here at Qubit. Thanks!

**Comment by kendrick-k** [comment]
ah sorry it s the refectoring option I suppose, me too, ok !

**Comment by kendrick-k** [comment]
commits have been squashed into one

**Comment by oliverwoodings** [comment]
Thanks for this!

**Pull request merged**